frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -167,7 +167,7 @@
         <tbody>
           <tr v-for="run in filteredRuns" :key="run.guid">
             <td>{{ time.formatTimestamp(run.time) }}</td>
-            <td>
+            <td name="guid">
               <acronym :title="run.guid">
```

Comment:
```suggestion
            <td>
              <acronym :title="run.guid" data-run-guid>
```

se prefiere el uso de [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) para encontrar cosas en las pruebas. esto se selecciona mediante `acronym[data-run-guid]`.